src/util/Util.js

```diff
@@ -198,15 +199,15 @@ define(['q', 'okta', './Logger', './Enums'], function (Q, Okta, Logger, Enums) {
     return explain;
   };
 
-  Util.introspectToken = function (authClient, widgetOptions) {
+  Util.introspectToken = function (widgetOptions) {
+    const domain = widgetOptions.baseUrl;
+    const stateHandle = widgetOptions.stateToken;
+    var trans = start({ domain, stateHandle });
     var deferred = Q.defer();
-    var trans = authClient.tx.introspect({
-      stateToken: widgetOptions.stateToken
-    });
     if (Q.isPromiseAlike(trans)) {
```

Comment:
This is still true, because otherwise the promise wont get resolved or rejected. Looking at [this](https://github.com/kriskowal/q/wiki/API-Reference#qispromisealikevalue) doc, looks like it just checks whether the given value is a promise.